src/python_testing/TC_OpstateCommon.py

```diff
@@ -1221,3 +1221,112 @@ async def TEST_TC_OPSTATE_BASE_2_5(self, endpoint=1):
             self.skip_step(20)
             self.skip_step(21)
             self.skip_step(22)
+
+    ############################
+    #   TEST CASE 2.6 - Optional Reports with DUT as Server
+    ############################
+    def steps_TC_OPSTATE_BASE_2_6(self) -> list[TestStep]:
```

Comment:
why is this in `Common` instead of being in TC_OPSTATE_2_6.py?
Why is opstate designed this way?

Questioning because we are in the process of separating out `test helper classes` in their own module so we do not rely on `import` working with sys.path having the current directory in it. Having these `common` scripts imported breaks that ... why did we do it?

Reply:
Rob did not write the original one, so I would ask @beckerzito (Lucas Becker). Rob is doing as in Rome in this PR

Reply:
So this was a designed proposed in 1.3. We were having a lot of redundant code in inherited clusters such as:
OperationalState:
- Oven operational state
- RVC operational state

ModeBase:
- Laundry Washer Mode
- Refrigerator Mode
- Microwave Oven Mode
- Oven Mode
- ....

It was getting hard to maintain the code. 
Basically all of them implement the same test script, only with different parameters. So we created "common" implementation based on the base cluster, and implemented an interface to parametrize each test for each cluster specification.

Reply:
@andy31415 It should definitely be possible for a TC_* test to have several local (in same directory) related helper modules, that are only relevant to a few tests, and not part of the general frameworking

Reply:
Issue created: https://github.com/project-chip/connectedhomeip/issues/34690